plugins/content/config/model.schema

```diff
@@ -291,6 +291,10 @@
     },
     "themeSettings": {
       "type": "object"
+    },
+    "_themePreset": {
```

Comment:
With this attribute I'm getting the classic error on the configuration settings page:
```console
Uncaught TypeError: constructorFn is not a constructor
    at child.createEditor (backbone-forms.js:842)
    at child.initialize (backbone-forms.js:803)
    at child.Backbone.View (backbone.js:991)
    at new child (backbone.js:1541)
    at child.createField (backbone-forms.js:136)
    at child.<anonymous> (backbone-forms.js:83)
    at Array.forEach (<anonymous>)
    at Function._.each._.forEach (underscore.js:79)
    at child.initialize (backbone-forms.js:81)
    at child.Backbone.View (backbone.js:991)
```

Reply:
Were you using the correct framework branch? https://github.com/adaptlearning/adapt_framework/issues/2360

Reply:
Unrelated – this branch currently utilises the core schema which is where it's picking up this attribute.

I would've thought it just needs adding to the [blacklist](https://github.com/CanStudios/adapt_authoring/blob/48d005e64a57627e9beff44b2911487321b839de/lib/contentmanager.js#L561-L586).